superset/db_engine_specs/TDengine.py

```diff
@@ -0,0 +1,36 @@
+# TDengine driver for Apache SuperSet
```

Comment:
This is redundant. Also, you need to include the license header in this file (you can just copy it from any other file).
```suggestion
```